### DISCUSSION TITLE
报错403

### DISCUSSION THREAD

**Discussion by soberkingomi** [pwh-pwh/aiwechat-vercel#15] [Q&A]
error, status code: 403, message: invalid character '<' looking for beginning of value
我应该都是配置好了的

**Comment by pwh-pwh** [comment]
你应该是GPT_URL配置错了，检查一下，配置成功访问域名/api/chat?msg=hi 看看结果

**Comment by soberkingomi** [comment]
> 你应该是GPT_URL配置错了，检查一下，配置成功访问域名/api/chat?msg=hi 看看结果

结果就是这个，我配置的是一个转发站的api > https://one.caifree.com/v1
我还试过配置为https://one.caifree.com ，结果一样的，每次改了环境变量我都redeploy的
/api/check的结果是 
```
spark: false
qwen: false
gemini: false
echo: true
gpt: true
DEFAULT BOT: gpt

**Reply by pwh-pwh** [reply]
/api/check检查逻辑是检查是否填写了对应参数，并不保证你填入的是否可以正常访问，所以要通过/api/chat检查

**Reply by soberkingomi** [reply]
![image](https://github.com/pwh-pwh/aiwechat-vercel/assets/76985627/6fb784a3-986c-4066-af5c-060f15591912)
这是我vercel的配置

**Comment by soberkingomi** [comment]
> /api/check检查逻辑是检查是否填写了对应参数，并不保证你填入的是否可以正常访问，所以要通过/api/chat检查

嗯嗯，但是/api/chat?msg=hi返回的结果就是最上面的403，接口没填错吧，需要加个v1吧

**Reply by pwh-pwh** [reply]
我的建议是本地调用一下你的gpt代理接口，看看能不能用

**Reply by pwh-pwh** [reply]
更大的可能性是你的这个代理站点不支持国外ip访问，建议试下其他的

**Reply by soberkingomi** [reply]
![image](https://github.com/pwh-pwh/aiwechat-vercel/assets/76985627/2b7e98f2-5c7a-41ad-b5c9-5a0ad7c60406)
![image](https://github.com/pwh-pwh/aiwechat-vercel/assets/76985627/a2668512-4064-495e-8040-8fae74066dc3)

本地是能调用的，挂梯调用也是国外ip

**Comment by soberkingomi** [comment]
> /api/check检查逻辑是检查是否填写了对应参数，并不保证你填入的是否可以正常访问，所以要通过/api/chat检查



> 更大的可能性是你的这个代理站点不支持国外ip访问，建议试下其他的

好的，我再试试，谢谢
